src/status.js

```diff
@@ -53,6 +37,63 @@ const ShowList = ((arr) => {
   });
 });
 
+const addToList = (() => {
+  const toDoTask = {
+    id: collection.length,
+    description: document.getElementById('todotitle').value,
+    completed: false,
+  };
+  collection.push(toDoTask);
+  localStorage.setItem('todoList', JSON.stringify(collection));
+  if (collection.length > 0) {
+    ShowList(collection);
+  }
+  form.reset();
+});
+
+const editTask = ((ev) => {
+  const buttonID = ev.target.id;
+  const dataGet = localStorage.getItem('todoList');
+  const data = JSON.parse(dataGet);
+  if (data) {
+    collection = data;
+  }
+  const btnId = collection[collection.findIndex((x) => x.id === parseInt(buttonID, 10))];
+  const last = collection.indexOf(btnId);
+  const test = ev.target.parentNode;
+  const editInput = test.querySelector('p');
+  editInput.contentEditable = true;
+  editInput.classList.add('test');
+  const a = test.querySelector('.fa-ellipsis-v');
+  a.style.display = 'none';
+  const b = test.querySelector('.fa-trash-o');
+  b.style.display = 'block';
+  collection[last].description = editInput.innerHTML;
+
+  editInput.addEventListener('keyup', () => {
+    collection[last].description = editInput.innerHTML;
+    localStorage.setItem('todoList', JSON.stringify(collection));
+  });
+});
+
+const removeTodo = ((ev) => {
+  const buttonId = ev.target.id;
+  collection = collection.filter(
+    (y) => y !== collection[collection.findIndex(
+      (x) => x.id === parseInt(buttonId, 10),
```

Comment:
Hi, thanks for your review. But in the project requirement, array index is array length. And array length started from 0 so that' why It starts from 0

Reply:
This issue still persist

Reply:
Hi, thanks for the review. But in the project requirement index is equal to array length so on the first iteration its length is zero so its index starts from zero. And its index is unique and updating when we remove something. I am confused about it.